src/components/album/AlbumChat.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import io from 'socket.io-client';
+
+const AlbumChat = ({ apiUrl }) => {
+  useEffect(() => {
+    const socket = io(process.env.REACT_APP_SOCKET_BASE_PATH);
+
+    // Subscribe to the album chat channel using albumId
+    const regexPattern = /\/albums\/([a-zA-Z0-9]+)/;
+    const match = apiUrl.match(regexPattern);
+    if (match) {
+      // Extract albumId from the URL
+      const albumId = match[1];
+      // Сonnection is established with the Socket.io server
+      socket.on('connect', () => {
+        // Request to the server to join the album chat
+        console.log('Connected to WebSocket server');
+        socket.emit('join:album_chat', albumId);
+      });
```

Comment:
Looking good! Last step to do based on ticket PS-184 is to add a step where you subscribe to the `chat:album` channel (In the ticket I previously had `chat:album:<albumId>` but that was incorrect, ticket has been fixed now). You'll want that nested here inside the \`on connect\`. For now the listener function for that could just log something to the console; we'll expand on it in the subsequent tickets